Firestore/Example/Tests/Integration/FSTDatastoreTests.mm

```diff
@@ -126,7 +127,7 @@ - (void)applyRemoteEvent:(FSTRemoteEvent *)remoteEvent {
   [expectation fulfill];
 }
 
-- (void)rejectListenWithTargetID:(FSTBoxedTargetID *)targetID error:(NSError *)error {
+- (void)rejectListenWithTargetID:(const TargetId)targetID error:(NSError *)error {
```

Comment:
BoxedTargetId exists because `NSDictionary` requires keys that extend from `NSObject`. This was put in the interface to avoid unboxing and reboxing the value across this API.

FSTBoxedTargetID is also potentially nullable, though in this usage it's not marked so, so these are equivalent.